pkg/ingester/ingester.go

```diff
@@ -564,6 +584,13 @@ func (i *Ingester) updateLoop(ctx context.Context) error {
 	localLimitMetricUpdateTicker := time.NewTicker(time.Second * 15)
 	defer localLimitMetricUpdateTicker.Stop()
 
+	var ownedSeriesUpdateTickerCh <-chan time.Time
```

Comment:
[nit] WDYT about merging `localLimitMetricUpdateTicker` and `ownedSeriesUpdateTickerCh`? We could simply call a generic `updateMetrics()` and then inside it we check if `i.cfg.UseIngesterOwnedSeriesForLimits || i.cfg.UpdateIngesterOwnedSeries`.